src/components/calcite-date-picker/calcite-date-picker.tsx

```diff
@@ -40,10 +40,10 @@ export class CalciteDatePicker {
   //
   //--------------------------------------------------------------------------
   /** Active range */
-  @Prop() activeRange?: "start" | "end" = "start";
+  @Prop() activeRange?: "start" | "end";
```

Comment:
For the default date-picker without an input, we don't want it to be in "start" or "end". We could have a new mode for both or just leave it empty.